source/how-tos/boot-zephyr.rst

```diff
@@ -0,0 +1,107 @@
+.. _boot-zephyr:
+
+Booting Zephyr
+-----------------
+
+We may follow below steps to setup Zephyr with |SPN| on |UP2| maker board and QEMU.
+
+|UP2| Board
+^^^^^^^^^^^^^^^^^
+
+1. Follow the '|Getting Started Guide|' to setup **Zephyr** build/dev environment
+
+.. |Getting Started Guide| raw:: html
+
+   <a href="https://docs.zephyrproject.org/latest/getting_started/getting_started.html" target="_blank">Getting Started Guide</a>
+
+2. Build an application uses synchronization sample application:
+
+  #. setup zephyr environment::
+  
+        source zephyr-env.sh
+        
+        or 
+        
+        zephyr-env.cmd
+
+  #. Build synchronization for |UP2| board::
+  
+        cd samples/synchronization
+        mkdir build_up2 && cd build_up2
+        cmake -GNinja -DBOARD=up_squared_sbl ..
+        ninja
+
+  #. Output file::
+        
+        samples/synchronization/build_up2/zephyr/zephyr.elf
+
+3. Build |SPN| image with zephyr.elf as a direct payload for simple test. 
+
+  - build |PN| for |UP2| board with the built zephyr.elf::
+  
+        python BuildLoader.py build apl -p <zephyr_root>/samples/synchronization/build_up2/zephyr/zephyr.elf
+
+
+4. Create a IFWI for |UP2|, follow the steps |here|.
+
+.. |here| raw:: html
+
+   <a href="https://slimbootloader.github.io/supported-hardware/up2.html#stitching" target="_blank">here</a>
+
+5. Flash IFWI and check the zephyr up & running thru serial console::
+
+    ......
+    Stage2 heap: 0xB4C000 (0x10C000 used, 0xA40000 free)
+    Payload ent***** Booting Zephyr OS zephyr-v1.13.0-1689-g4a7c422182 *****
+    threadA: Hello World from up_squared!
+    threadB: Hello World from up_squared!
+    threadA: Hello World from up_squared!
+    threadB: Hello World from up_squared!
+    threadA: Hello World from up_squared!
+
+
+QEMU
+^^^^^^^^^
+
+1. Build an application uses synchronization sample application:
+  
+  #. setup zephyr environment::
+  
+        source zephyr-env.sh
+        
+        or 
+        
+        zephyr-env.cmd
+
+  #. Build synchronization for QEMU::
+
+        cd samples/synchronization
+        mkdir build_qemu && cd build_qemu
+        cmake -GNinja -DBOARD=qemu_x86 ..
+        ninja
+
+  #. Output file::
+
+        samples/synchronization/build_qemu/zephyr/zephyr.elf
+
+2. Build |SPN| image with zephyr.elf as a direct payload for simple test. 
```

Comment:
- Quote ``zephyr.elf`` this way
- 'as payload' instead of 'as a direct payloader'